src/encoding_implementations/ordinal_pattern.jl

```diff
@@ -29,16 +32,28 @@ julia> x = [4.0, 1.0, 9.0];
 
 julia> c = OrdinalPatternEncoding(3);
 
-julia> encode(c, x)
+julia> i = encode(c, x)
 3
 
-julia> decode(c, 1)
+julia> decode(c, i)
 3-element SVector{3, Int64} with indices SOneTo(3):
  2
  1
  3
 ```
 
+If you want to encode something that is already a permutation pattern, then you
+can use the non-exported `permutation_to_integer` function. Using [`encode`](@ref)
+directly will do a double-call to `sortperm`, which destroys the
+one-to-one mapping between encoded integers and decoded permutation patterns.
+
+```julia
+using ComplexityMeasures
+p = sortperm([4.0, 1.0, 9.0])
+c = ComplexityMeasures.OrdinalPatternEncoding(length(p));
+ComplexityMeasures.permutation_to_integer(c, p)
+```
+
```

Comment:
```suggestion
If you want to encode something that is already a permutation pattern, then you
can use the non-exported `permutation_to_integer` function.
```

I think we should keep docstrings small and not say much about such advanced situations.